Source/Core/Core/PerformanceSampleAggregator.h

```diff
@@ -21,21 +23,33 @@ struct CompletedReport
 class PerformanceSampleAggregator
 {
 public:
-  static std::chrono::microseconds GetInitialSamplingStartTimestamp();
-  static std::chrono::microseconds GetRepeatSamplingStartTimestamp();
-  static std::chrono::microseconds GetCurrentMicroseconds();
-  static CompletedReport GetReportFromSamples(const std::vector<PerformanceSample>& samples);
+  // Called on game start / title switch.
```

Comment:
Could we reduce the amount of static state? It feels like some of those (what used to be member variables in DolphinAnalytics) shouldn't be static.

Reply:
I think I might not have been clear enough in my previous comment. To clarify:

* s_samples, s_next_starting_timestamp, InitializePerformanceSampling, AddSampleIfSamplingInProgress, PopReportIfComplete seem like they can be made non-static
* DolphinAnalytics should probably have PerformanceSampleAggregator as a member variable.

Reply:
Done.  The other variables were only static because PerformanceSampleAggregator was too, but putting it into DolphinAnalytics fixes that problem.